templates/bundles/PrestaShopBundle/Admin/Common/Grid/Columns/Content/date_time.html.twig

```diff
@@ -23,10 +23,8 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  *#}
 
-{% set valueToDisplay = record[column.id]|date(column.options.format) %}
-
 {% if record[column.id] == constant('PrestaShop\\PrestaShop\\Core\\Util\\DateTime\\DateTime::NULL_VALUE') %}
-  {% set valueToDisplay = column.options.empty_data %}
+  {{  column.options.empty_data }}
+{% else %}
+  <time datetime="{{ record[column.id]|date(constant('PrestaShop\\PrestaShop\\Core\\Util\\DateTime\\DateTime::ISO_DATETIME_FORMAT')) }}">{{ record[column.id]|date(column.options.format) }}</time>
```

Comment:
```suggestion
  <time datetime="{{ record[column.id]|date(constant('PrestaShop\\PrestaShop\\Core\\Util\\DateTime\\DateTime::ISO_DATETIME_FORMAT')) }}">{{ record[column.id]|date(column.options.format) }}</time>
```

The time element is not supported at all on IE, this means that date fields will be totally empty on IE :/

I'm not in favor of supporting IE anymore, but this should be decided and communicated before using non-supported features, wdyt?

Reply:
```
<!--[if IE]>please, update it finally 😂 <![endif]-->
```

Reply:
IE is dead, if it works on edge, that should be enough as we target the BO and not the FO